site/docs/numbers/porting/lnpChecker.mdx

```diff
@@ -0,0 +1,191 @@
+---
+id: lnpChecker
+title: Check number(s) portability
+slug: /numbers/guides/porting/lnpChecker
+description: How to check number(s) portability
+keywords:
+  - bandwidth
+  - numbers
+  - porting
+  - portin
+  - port-in
+  - port
+  - lnp
+  - portability
+  - checker
+  - check
+image: ../../../static/img/bandwidth-logo.png
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const accountId = "{accountId}";
+export const orderId = "{orderId}";
+
+export const Highlight = ({children, color}) => (
+    <span
+      style={{
+        backgroundColor: color,
+        borderRadius: '2px',
+        color: '#fff',
+        padding: '0.2rem',
+      }}>
+      {children}
+    </span>
+  );
+
+A number or set of numbers can be checked to see if they can be ported into the Bandwidth Network.
+:::info
+As an alternative to individual porting the `/bulkPortins` API eliminates the need for the `/lnpchecker` API by sorting the list of TNs automatically into a set of port-in requests by Port Type. For more details follow the [How to Bulk Port](/docs/numbers/guides/porting/createBulkPortins).
+:::
+
+TNs can be ported in a single `/portins` request if all the following are true:
+
+1. They all have the same Port Type (see the `Port Types` section below)
+2. Telephone numbers all have the same losing carrier (toll free telephone numbers all have the same losing RespOrg)
+3. They are all associated with the same billing TN and Service Address
+
+:::caution
```

Comment:
Please move line 49 - line 58 (caution box) to line 187 (after the code examples)